### ISSUE TITLE
Get current time from video player

### ISSUE THREAD

**Issue by MeshkaniMohammad** [MrJai/flutter_to_airplay#11]
Hi.
Could you make it in a way to get the current time and the total time of video or make it in a way to sync with other video player plugins?

**Comment by vanlooverenkoen** [comment]
I don't think that is something that needs to be done by this plugin. I am using `video_player` and it worked like a charm. Everything is reported correctly as long as you did the initial `video_player` configuration correctly.
